.github/workflows/sync-subnet-evm-branch.yml

```diff
@@ -17,4 +17,49 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version: "~1.21.12"
-          check-latest: true
\ No newline at end of file
+          check-latest: true
+
+      - name: Add Remote
+        run: |
+          git remote add subnet-evm https://github.com/ava-labs/subnet-evm
+    
+      # TODO: Replace test branch with actual (eg, coreth)
+      - name: Fetch Remote Branch (${{ github.event.inputs.remoteBranch }})
+        id: fetch_remote_branch
+        run: |
+          git fetch subnet-evm ${{ github.event.inputs.remoteBranch }}
+          git fetch subnet-evm coreth-diff-script
+          result=$(git rev-parse --short=8 subnet-evm/${{ github.event.inputs.remoteBranch }})
+          echo result="${result}" >> "${GITHUB_OUTPUT}"
+
+      # TODO: Remove this step when the script is checked in
+      - name: Checkout script
+        run: |
+          git checkout subnet-evm/coreth-diff-script -- ./scripts/apply_diff_and_rename.sh
+
+      - name: Apply diff
+        run: |
+          ./scripts/apply_diff_and_rename.sh subnet-evm/${{ github.event.inputs.remoteBranch }}
+
+      # TODO: Should this step be removed?
+      - name: Restore .github/workflows
+        run: |
+          git checkout -- .github/workflows
```

Comment:
AFAIK it requires a special token if you want to override a workflow with a workflow